gate_fixtures/invoice_store.py

```
@@ -0,0 +1,41 @@
+
+
+def delete_invoices(customer_id):
+    conn = connect()
+    conn.cursor().execute(
+        "DELETE FROM invoices WHERE customer_id = '%s'" % customer_id
+    )
+    conn.commit()
+
+
+def invoice_token(invoice_id, secret):
+    return hashlib.md5((str(invoice_id) + secret).encode()).hexdigest()
```

Ruff (0.16.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)